CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## HEAD
 
+- Breaking: Support for Ruby 2.2 is soft removed going forward only Ruby versions on the currently relased stack will be supported
```

Comment:
Is it worth raising `required_ruby_version` in `hatchet.gemspec` to the lowest tested version? Since:
- IMO it's misleading for the minimum version to be set to something that isn't tested
- It's pretty easy to upgrade Ruby in the CI workflows that run Hatchet
- Anyone who really really needs older Ruby (unlikely) can stick on an older Hatchet gem version (and in reality, if they aren't updating Ruby, they aren't updated Hatchet either)

Reply:
It's a little more complicated. Other gems depend on our gem like https://rubygems.org/gems/platform-api/versions/3.5.0. If we set a minimum version, we force them into a minimum version. I would like to announce the intention that we will rev minimum required, but I've not thought long and hard enough about it to be ready to make the call.

Reply:
Ah I didn't know there were non-buildpack usages - happy to defer for now :-)